src/universal/config/app.ts

```diff
@@ -33,7 +33,7 @@ export const FeatureToggle = {
   bekendmakingenDatasetActive: true,
   evenementenDatasetActive: false,
   klachtenActive: true,
-  bezwarenActive: !IS_AP,
+  bezwarenActive: false,
```

Comment:
Is dit een bewuste aanpassing?

Reply:
Ja, als we er weer mee aan de gang gaan kan het weer terug gezet worden.